### ISSUE TITLE
Create readout file for datacutr

### ISSUE THREAD

**Issue by barnett11** [pharmaverse/datacutr#107]
Create a readout file to include,
Domains run through datacutr, and approach taken, including date variable chosen (? From function so not needed?)
Summary of records removed per domain (or for DM, changed)

**Comment by barnett11** [comment]
Potentially split into two parts given differences in modular vs wrapped approach

**Comment by barnett11** [comment]
Proposal for review:
A readme file for each function patient_cut, date_cut and special_dm_cut that indicates number of records removed (or changed) per domain. 
One readme template as reference?
@kieranjmartin do you have any opinion or recommendation please? Thanks

**Comment by kieranjmartin** [comment]
With this kind of information, is this intended to be saved somewhere? That is we expect this to be part of a report that is submitted somewhere?

The reason I ask is just to check that a physical report is definitely needed, as we can return this sort of information in session. 

I don't necessarily have a specific best practice here; you could even support different methods. E.g. `export_md`, `export_pdf` or `export(format= "pdf")`. 

Basically I would design towards whatever you think the most useful output format is, but allow flexibility for users to create their own outputs if they really want to
